openhtf/core/test_state.py

```diff
@@ -235,11 +235,8 @@ def set_status_from_phase_outcome(self, phase_outcome):
           self.is_finalized, phase_outcome)
     return self.is_finalized
 
-  def mark_test_started(self, dut_id):
-    """Set the TestRecord's dut_id and start_time_millis fields."""
-    assert self._status == self.Status.WAITING_FOR_TEST_START
```

Comment:
maybe assert that `start_time_millis is None`?  this assert was here to make sure we didn't accidentally blow away a previous `start_time_millis`, as that would be a particularly difficult bug to track down and diagnose.